settings.gradle

```diff
@@ -1,3 +1,23 @@
+pluginManagement {
+    resolutionStrategy {
+        eachPlugin {
+            // Work around https://github.com/gradle/gradle/issues/1697.
+            if (requested.version == null) {
+                def versionPropertyName = (requested.id.id == 'org.jetbrains.kotlin.jvm') ?
+                        "kotlinVersion" : "${requested.id.name}Version"
```

Comment:
Why is this special case for Kotlin necessary, what is `request.id.name` for the Kotlin plugin?

Reply:
It would be just `jvm`, i.e. the last part after the dot.

Reply:
Ok, that would be ugly.